content/source/docs/enterprise/getting-started/access.html.md

```diff
@@ -44,13 +44,13 @@ To collaborate with your colleagues in TFE, you'll all need access to the same T
 
 First, navigate to the settings page for your organization — you can reach it from the "Settings" link found at the top of every page. Once there, click the "Teams" link in the sidebar navigation.
 
-The list of teams starts with just one team, named "owners." Don't add users to this team yet; instead, enter a new team name (like "core-infrastructure") in the text field and click the "Create team" button.
+The list of teams starts with just one team, named "owners." Don't add users to this team yet; instead, enter a new team name (like "core-infrastructure") in the text field and click the "Create team" button. This will take you to the new team's settings page:
 
-![adding members to a team](./images/access-add-members.png)
+![adding members to a team](../users-teams-organizations/images/teams-team-settings.png)
 
 Add as many users as you'd like by typing their TFE username in the text field and clicking "Add user." Added users won't receive a notification, but your organization will be available the next time they access TFE.
```

Comment:
I'd prefer `"Add member".` - the member change needs to be made to match the screenshot, and the period/quote inversion since there's no period in the button text, but I know we've wrangled about that style before so go with what you prefer on that. :)